src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -62,16 +62,26 @@
 
     private static final ExecutorService executorService = Executors.newCachedThreadPool();
 
-    /** Thread safe formatter for date. */
+    /**
+     * Thread safe formatter for date.
+     */
     private static final FastDateFormat DATE_FORMATTER = FastDateFormat.getInstance("yyyy-MM-dd HH:mm:ss z");
 
     private static final Gson gson = new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.IDENTITY).create();
 
-    public static void sendBuildStaredNotification(Run run, TaskListener listener, boolean isFromPrebuild) {
+    private final Run run;
+    private final TaskListener listener;
+
+    public Office365ConnectorWebhookNotifier(Run run, TaskListener listener) {
```

Comment:
by this Run and TaskListener are not passed over and over from one method to another